content/integrations/faq/open-file-descriptors-metric-collection.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Process open file descriptors metric collection
+kind: faq
+further_reading:
+- link: "/integrations/process"
+  tag: "Documentation"
+  text: Learn more about the Datadog Process integrations
+---
+
+For the [Datadog - process integration](/integrations/process/) `system.processes.open_file_descriptors` metric is only available for processes running as the same user as the agent (dd-agent).
+
+If you wish to collect this metric, a workaround may be to run a small Python script as root that uses psutil to read this metric and sends it to the Datadog Agent using [DogStatsD](/developers/dogstatsd).
+
+```python
+#!/opt/datadog-agent/embedded/bin/python
+
+import psutil
+import socket
+import sys
+
+process_filter = sys.argv[1]
+
+for proc in psutil.process_iter():
+    if process_filter in proc.name():
+        payload = "open_file_descriptors:{0}|g|#process_name:{1},pid:{2}".format(
+            proc.num_fds(), proc.name(), proc.pid)
+        sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM) # UDP
```

Comment:
Does the `UDP` stand for `user datagram protocol`? I wasn't sure what the comment was for in the script.